public/index.html

```diff
@@ -5,7 +5,7 @@
   <link href="css/index.css" type="text/css" rel="stylesheet">
 </head>
 
-<body>
+<body class="body">
```

Comment:
This is bad practice, instead of adding a unique class with the same name as the body tag, just select the body tag directly inside the css. Like this:
```html
<body>
Hello World
</body>
```
```css
body {
  background: black;
  color: white;
}
```